pandas/tests/extension/base/getitem.py

```diff
@@ -134,12 +134,42 @@ def test_take(self, data, na_value, na_cmp):
 
```

Comment:
The test above (can't comment on the exact line), should actually fail now, as it is still testing the old behaviour (-1 giving NaN). So need to edit the test to check for `data[len(data)-1]` instead of `nvalue`